azerothcore/azerothcore-wotlk
# fix(scripts/Ulduar): Spinning UP during 4th phase

Fixed #1611.





## Changes Proposed:
-  
-  


## Issues Addressed:
- Closes #1611



## Tests Performed:
- Tested ingame - works okay.
-



## How to Test the Changes:

1. `.go creature id 33350`
2.  Aggro Mimiron
3.  Go to phase 4.
4.  Observe Spinning Up.

## Known Issues and TODO List:



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
